Automattic/wp-calypso
# Hosting: Add a new Miscellaneous Card for clearing cache

#### Changes proposed in this Pull Request

We want other admin stuff to live here.
One example is the Clear Cache action.

We want to allow users to be able to clear cache
if an HE has enabled this ability for them.

#### Testing instructions

Calypso:
* Create an Atomic site
* Go to Hosting Configuration
* See Miscellaneous Card
* See Clear Cache button
* Click it
* Enter reason
* Check logstash for logs

#### New card looks

![Screenshot from 2020-04-16 15-04-23](https://user-images.githubusercontent.com/1103398/79459512-acbbf100-7ff3-11ea-8b3e-9299b9fdde41.png)

and the reason popup.

![Screenshot from 2020-04-16 15-04-52](https://user-images.githubusercontent.com/1103398/79459526-b180a500-7ff3-11ea-822c-3c1f82e0c8a3.png)
